modules/migration-compatibility-guidelines.adoc

```diff
@@ -26,14 +26,8 @@ Installed manually with the `operator-3.7.yml` file.
 
 Installed manually with the `operator.yml` file.
 
-|4.6 and later versions |1.5.1 |{mtc-full} Operator.
+|4.6 and later |1.6.x^[1]^ |{mtc-full} Operator.
 
 Installed with the Operator Lifecycle Manager.
 |===
-
-// For OCP 4.9 and later
-// |4.9 |1.6.x^[1]^ |{mtc-full} Operator.
-//
-// Installed with the Operator Lifecycle Manager.
-// |===
-// ^1^ Latest z-stream release.
+^1^ Latest z-stream release.
```

Comment:
Note that our build process doesn't transform this so that the footnote is grouped with the table. It still looks okay visually, but is not the most accessible thing. Might want to consider another way of presenting this when you're not crunched for time.

Reply:
We do this in other place: https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virtual_disks/virt-features-for-storage.html

Reply:
I'd make the same comment if I were reviewing that table. It's just an FYI. :shrug: